osm2gtfs/core/osm_connector.py

```diff
@@ -520,31 +520,31 @@ def _build_station(self, stop_area, osm_type):
                 else:
                     sys.stderr.write(
                         "Error: Station member was not found in data")
-                    sys.stderr.write("https://osm.org/relation/" +
+                    sys.stderr.write(" https://osm.org/relation/" +
                                      str(stop_area.id) + "\n")
-                    sys.stderr.write("https://osm.org/node/" +
+                    sys.stderr.write(" https://osm.org/node/" +
                                      str(member.ref) + "\n")
         if len(members) < 1:
             # Stop areas with only one stop, are not stations they just
             # group different elements of one stop together.
             sys.stderr.write(
                 "Error: Station with no members has been discarted:\n")
-            sys.stderr.write("https://osm.org/relation/" +
+            sys.stderr.write(" https://osm.org/relation/" +
                              str(stop_area.id) + "\n")
             return None
 
         elif len(members) == 1:
             sys.stderr.write(
-                "Warning: Station has only one platform and is discarted\n")
-            sys.stderr.write("https://osm.org/relation/" +
+                "Note: Stop area has only one platform and won't be a station \n")
```

Comment:
This is not very easy to understand.
suggestion: `OSM stop area has only one platform and cannot be used as a GTFS station`